### PR TITLE
test: cypress - switch widget automation

### PR DESCRIPTION
**Pull request by sharanya-appsmith**
In deploy and preview mode
1. Validate general properties - Default state values set via JS
2. Validate general properties - Visible values set via JS
3. Validate general properties - Disabled values set via JS
4. Validate error texts on unacceptable values in property pane
5. Validate alert on onChange

### CODE REVIEW

File: app/client/cypress/locators/Widgets.json [hunk 1]
```diff
@@ -201,6 +202,7 @@
   "codescannerwidget": ".t--widget-codescannerwidget",
   "widgetNameSpan": ".t--widget-propertypane-toggle > .t--widget-name",
   "listWidgetv2": ".t--widget-listwidgetv2",
+  "dropdownMenuItem":".menu-item-text",
```
Comment (NandanAnantharamu) [inline app/client/cypress/locators/Widgets.json:205]: This locator is there in Table.ts please use that

File: app/client/cypress/locators/Widgets.json [hunk 2]
```diff
@@ -79,6 +79,7 @@
   "tableOnRowSelect": ".t--property-control-onrowselected .t--open-dropdown-Select-Action",
   "switchInput": ".t--draggable-switchwidget span.t--widget-name",
   "switchLabel": ".t--draggable-switchwidget label",
+  "switch":".bp3-switch",
```
Comment (NandanAnantharamu) [inline app/client/cypress/locators/Widgets.json:82]: Can you move this to ts helper methods, all new locators to be added to ts helpers

Comment (sharanya-appsmith) [reply app/client/cypress/locators/Widgets.json:82]: Done. Removed.

File: app/client/cypress/locators/Widgets.json (hunk 1 above)
Comment (sharanya-appsmith) [reply app/client/cypress/locators/Widgets.json]: Done.

**Review by NandanAnantharamu — APPROVED** [review]
lgtm

**Review by NandanAnantharamu — APPROVED** [review]
LGTM

**Pull request merged**